src/input/code_templates/c/static_code/exi_basetypes_decoder.c.jinja

```diff
@@ -341,6 +346,22 @@ int exi_basetypes_decoder_integer_64(exi_bitstream_t* stream, int64_t* value)
     return error;
 }
 
+int exi_basetypes_decoder_signed(exi_bitstream_t* stream, exi_signed_t* value)
+{
+    int sign;
+    int error;
+
+    error = exi_basetypes_decoder_bool(stream, &sign);
```

Comment:
sign should be initialised.
Maybe you can change this code snippet to:
```
int sign = 0;

int error =  exi_basetypes_decoder_bool(stream, &sign);
```

Reply:
Do you want me to init the other values in the basetypes de/encoder.c?

Reply:
Thx for pointing that out. But I think init the other values should be in another PR.